site/source/docs/debugging/Sanitizers.rst

```diff
@@ -330,3 +330,38 @@ can cause traps. Hence, it is not enabled by default.
         (longjmp and C++ exceptions *are* supported)
   SUMMARY: AddressSanitizer: stack-use-after-return (a.out.wasm+0xd8f)
   ...
+
+Configuration
+-------------
+
+ASan can be configured via a ``--pre-js`` file:
+
+.. code-block:: javascript
+
+  Module.ASAN_OPTIONS = 'option1=a:option2=b';
+
+For example, put the above snippet with your options into ``asan_options.js``,
+and compile with ``--pre-js asan_options.js``.
+
+For standalone LSan, use ``Module.LSAN_OPTIONS`` instead.
+
+For a detailed understanding of the flags, see the `ASan documentation`__.
+
+__ https://github.com/google/sanitizers/wiki/AddressSanitizerFlags#run-time-flags
```

Comment:
It might be worth adding a disclaimer that in general custom settings are not guaranteed to work because we have no test coverage for them.